<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1586165818033' stop='1586165821776'>
    <name>Demo Test case</name>
    <title>Demo Test case</title>
    <test-cases>
        <test-case start='1586165818033' status='failed' stop='1586165821192'>
            <name>addition test</name>
            <title>addition test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: No element found using locator: by.cssContainingText(".ng-binding", "13")</message>
                <stack-trace>NoSuchElementError: No element found using locator: by.cssContainingText(".ng-binding", "13")
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27
                    at ManagedPromise.invokeCallback_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as getText] (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as getText] (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at homepage.verifyResult (D:\IVL\AutoDocs\Protractor\Projects\ProtractorDemo\pages\homepage.js:25:23)
                    at UserContext.&lt;anonymous> (D:\IVL\AutoDocs\Protractor\Projects\ProtractorDemo\tests\calculator.js:27:18)
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                From: Task: Run it("addition test") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\IVL\AutoDocs\Protractor\Projects\ProtractorDemo\tests\calculator.js:7:5)
                    at addSpecsToSuite (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\IVL\AutoDocs\Protractor\Projects\ProtractorDemo\tests\calculator.js:5:1)
                    at Module._compile (internal/modules/cjs/loader.js:1158:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
                    at Module.load (internal/modules/cjs/loader.js:1002:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:901:14)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1586165821192' status='failed' stop='1586165821775'>
            <name>substraction test</name>
            <title>substraction test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>ReferenceError: fs is not defined</message>
                <stack-trace>ReferenceError: fs is not defined
                    at D:\IVL\AutoDocs\Protractor\Projects\ProtractorDemo\conf\conf.js:88:28
                    at ManagedPromise.invokeCallback_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\abhishek.mishra\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>